Example/AppDelegate.swift

```diff
@@ -25,7 +25,7 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 	]
 
 	private lazy var settingsWindowController = SettingsWindowController(
-		preferencePanes: settings,
+		settingsPanes: settings,
```

Comment:
Per the issue, this should simply be `panes`.

Reply:
Done